ChartsDemo-iOS/Swift/Demos/PieChartViewController.swift

```diff
@@ -21,7 +21,7 @@ class PieChartViewController: DemoBaseViewController {
         super.viewDidLoad()
         
         // Do any additional setup after loading the view.
-        self.title = "Half Pie Bar Chart"
+        self.title = "Pie Chart"
```

Comment:
I think we should keep the 'half' word?

Reply:
I think that the "Pie Chart" is correct since the controller name is "PieChartViewController".
BTW, I have fixed the title to "Half Pie Chart" from  "Half Pie Bar Chart" for "HalfPieChartViewController.swift"

Reply:
I see.